homework/index.js

```diff
@@ -30,18 +30,94 @@
     return elem;
   }
 
+  function render(parent, repositories) {
+    const table = createAndAppend('table', parent, {});
+    const tbody = createAndAppend('tbody', table, {});
```

Comment:
If no need for passing anything no need for an extra Braces while it's assumed by default on line 19